layers/+tools/seeing-is-believing/packages.el

```diff
@@ -0,0 +1,33 @@
+;;; packages.el --- seeing-is-believing Layer packages File for Spacemacs
+;;
+;; Copyright (c) 2012-2014 Sylvain Benner
+;; Copyright (c) 2015 Brandon Conway & Contributors
+;;
+;; Author: Brandon Conway <brandoncc@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(setq seeing-is-believing-packages
+    '(
+       seeing-is-believing
+      ))
+
+
+(defun seeing-is-believing/init-seeing-is-believing ()
+  (use-package seeing-is-believing
+    :defer t
+    :init
+    (progn
+      (require 'seeing-is-believing)
+      (add-hook 'ruby-mode-hook 'seeing-is-believing))
```

Comment:
If you require `seeing-is-believing` in the `:init` section, you are essentially making it not deferred and loaded at startup every time.